app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/charge_item.scala.html

```diff
@@ -44,11 +44,11 @@
             case (Some(paymentMethod), _) =>
                     makePaymentDetailsHtml("paymentMethod", paymentMethod.toString, payment.amount.inPounds(), payment.clearingDate)
             case (None, Some(clearingReason)) => clearingReason match {
-                case IncomingPayment | OutgoingPayment | WriteOff | Reversal | MassWriteOff | AutomaticClearing | SomeOtherClearingReason =>
+                case IncomingPayment | OutgoingPayment | WriteOff | Reversal | MassWriteOff | AutomaticClearing =>
                     makePaymentDetailsHtml("clearingReason", clearingReason.toString, payment.amount.inPounds(), payment.clearingDate)
-                case InvalidClearingReason if (payment.method.isDefined) =>
+                case SomeOtherClearingReason if (payment.method.isDefined) =>
                     makePaymentDetailsHtml("paymentMethod", payment.method.fold("")(_.toString), payment.amount.inPounds(), payment.clearingDate)
-                case InvalidClearingReason if (!payment.method.isDefined) =>
+                case SomeOtherClearingReason if (!payment.method.isDefined) =>
```

Comment:
do you even need this case now? Couldn't you just have:
```scala
case (Some(paymentMethod), _) => ...
case (_, Some(clearingReason)) =>
  makePaymentDetailsHtml("clearingReason", clearingReason.toString, payment.amount.inPounds()
```
the order of the pattern match means you don't need to check if `paymentMethod` is defined or not after that first case

Reply:
looking at it again, the logic is now incorrect - let me fix it